src/components/OfflineWithFeedback.tsx

```diff
@@ -18,7 +18,7 @@ import MessagesRow from './MessagesRow';
 
 type OfflineWithFeedbackProps = ChildrenProps & {
     /** The type of action that's pending  */
-    pendingAction: OnyxCommon.PendingAction;
+    pendingAction?: OnyxCommon.PendingAction;
```

Comment:
Can you test what will happen when `pendingAction` will be undefined?

Reply:
Nothing happened - all works good